### DISCUSSION TITLE
Fix spelling grammar is way too chatty

### DISCUSSION THREAD

**Discussion by gerroon** [pfrankov/obsidian-local-gpt#42] [Q&A]
Hi

Thanks for the plugin. I am now using it to fix grammar stuff, it generally works except that it keeps giving me explanations of the changes which I don't want, because then I need to keep cleaning up.

Is there a way to make it so that it only gives me the result nothing else? Maybe there is also a better model for this

I use it with Ollama, llama3.1

Here is my edited prompts

**system:** "You are an AI assistant that follows instruction extremely well. Help as much as you can and never provide explanations in the final results"

**prompt:** "Proofread the below for spelling and grammar, stay close to the original, never provide explanations "

**Comment by pfrankov** [comment]
It all depends on model you use. There is no universal prompt for every model that will force it to do what you ask to do.

**Reply by gerroon** [reply]
I understand that, that is why I listed the model I use with it.
